### DISCUSSION TITLE
Configuration - onpremises with DSL editor

### DISCUSSION THREAD

**Discussion by tigeressdress** [structurizr/onpremises#141] [General]
I have recently downloaded the onpremises version, but I would like to use the DSL editor. It says it needs to be enabled - image attached - is what i see and nothing else. 
I have looked at the src - https://github.com/structurizr/java/tree/master/structurizr-dsl ,  but doesn't actually say - how to configure this and upload into the image or a container.
appreciate your assistance.
thanks

![structurizr](https://github.com/user-attachments/assets/256b2e20-0a54-4072-906f-6b0d30701539)

**Comment by simonbrowndotje** [comment]
See https://docs.structurizr.com/onpremises/configuration ... specifically the `structurizr.feature.ui.dslEditor` property.
